examples/with-svelte/packages/ui/tsconfig.json

```diff
@@ -1,15 +1,10 @@
 {
-  "extends": "@repo/typescript-config/svelte.json",
+  "extends": ["@repo/typescript-config/svelte.json", "./.svelte-kit/tsconfig.json"],
```

Comment:
`.svelte-kit` only gets generated after a build is run so running `check-types` before building will result in failure